bulk_insert_test.go

```diff
@@ -113,6 +113,59 @@ func Test_insertObject(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func Test_insertObjSetWithCallback(t *testing.T) {
+	type Table struct {
+		ID            uint `gorm:"primary_key;auto_increment"`
+		RegularColumn string
+		Custom        string `gorm:"column:ThisIsCamelCase"`
+	}
```

Comment:
The test case this was coped from was to test the `gorm:"column"` tag. I think it makes the test unnecessary complex to not just use a simple tags and names as possible.

Reply:
Sorry, i have no idea how to test it with `github.com/DATA-DOG/go-sqlmock`, i never use this packge. but i know that it works with my postgres database. would u plz help me for testing it after merging into epic branch?

Reply:
Sure thing, I can write tests and push to this branch before merging as well, that's probably better. I'll hold off doing that until @t-tiger approves this PR so we know it's the final draft.